api/app/utils/strings.py

```diff
@@ -62,3 +62,47 @@ def build_response_email_body(answers, language, application_form):
                 stringified_summary += '{question}\n{answer}\n\n'.format(question=question_translation.headline, answer=answer_value)
 
     return stringified_summary
+
+
+def build_response_html_app_info(response, language):
+    """
+    Stringifying the dictionary application information, for output in a html file, with the response_id and applicant name contact info as
+     paragraphs (<p>)
+    """
+    
+    stringified_app_info = f"<p><b> Response ID:<b> {response.id}<p> <p><b>Full name:<b> {response.user.firstname} {response.user.lastname}<p>"
```

Comment:
The closing tags don't seem correct here? Should be `</b>` and `</p>`  and `</title>` below.